keras_cv/layers/preprocessing/random_flip.py

```diff
@@ -32,34 +32,42 @@
 
 
 @keras.utils.register_keras_serializable(package="keras_cv")
-class RandomFlip(BaseImageAugmentationLayer):
+class RandomFlip(VectorizedBaseImageAugmentationLayer):
     """A preprocessing layer which randomly flips images.
 
     This layer will flip the images horizontally and or vertically based on the
     `mode` attribute.
 
     Input shape:
-      3D (unbatched) or 4D (batched) tensor with shape:
-      `(..., height, width, channels)`, in `"channels_last"` format.
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format.
 
     Output shape:
-      3D (unbatched) or 4D (batched) tensor with shape:
-      `(..., height, width, channels)`, in `"channels_last"` format.
-
-    Arguments:
-      mode: String indicating which flip mode to use. Can be `"horizontal"`,
-        `"vertical"`, or `"horizontal_and_vertical"`, defaults to
-        `"horizontal"`. `"horizontal"` is a left-right flip and `"vertical"` is
-        a top-bottom flip.
-      seed: Integer. Used to create a random seed.
-      bounding_box_format: The format of bounding boxes of input dataset.
-        Refer to
-        https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
-        for more details on supported bounding box formats.
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format.
+
+    Args:
+        mode: String indicating which flip mode to use. Can be `"horizontal"`,
+            `"vertical"`, or `"horizontal_and_vertical"`, defaults to
+            `"horizontal"`. `"horizontal"` is a left-right flip and
+            `"vertical"` is a top-bottom flip.
+        rate: A float that controls the frequency of flipping. 1.0 indicates
+            that images are always flipped. 0.0 indicates no flipping.
+            Defaults to 0.5.
+        seed: Integer. Used to create a random seed.
+        bounding_box_format: The format of bounding boxes of input dataset.
+            Refer to
+            https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
+            for more details on supported bounding box formats.
```

Comment:
This is going to cause lint errors post-merge (as the line length of the longest line is >80)

Can you please add `# noqa: E501` to the end of this docstring?

Reply:
Should be resolved.
(I didn't see any error by executing `./shell/lint.sh` before the modification. Why?)

Reply:
Probably because you haven't merged from master since we updated the linter to enforce this. Thanks for the fix! Should be good now.